test-requirements.txt

```diff
@@ -1,2 +1,3 @@
 nose
 mock
+flask
```

Comment:
I do feel strongly about pinning an upper bound. I don't want the tests to break because of flask code changes.